Data/fsl_default/nidm.json

```diff
@@ -85,608 +85,623 @@
-      "dct:format": "image/nifti",
-      "nfo:fileName": "Contrast.nii.gz",
+      "crypto_sha512": "f2df3de73c023be1d8d1f73703e3ffdbf08b69a27ae304cdf053e8639da9e579fd21a52a66584e04c865239f408bd30d7b5e690985ce7a20ccd2a50c3306fd55",
+      "dct_format": "image/nifti",
+      "nfo_fileName": "Contrast.nii.gz",
       "nidm_contrastName": "tone counting vs baseline",
       "nidm_inCoordinateSpace": {
-        "@id": "niiri:b8f4de59-d839-4f91-86a7-2697c1f65f44"
+        "x_id": "niiri:b8f4de59-d839-4f91-86a7-2697c1f65f44"
       },
-      "prov:atLocation": {
-        "@type": "xsd:anyURI",
-        "@value": "http://neurovault.org/collections/BNRPBNVV/fsl_default.nidm/Contrast.nii.gz"
+      "prov_atLocation": {
+        "x_type": "xsd:anyURI",
+        "x_value": "http://neurovault.org/collections/BNRPBNVV/fsl_default.nidm/Contrast.nii.gz"
       },
-      "prov:wasGeneratedBy": {
-        "@id": "niiri:6f6a8d7b-1697-418f-8286-b84f22325b5b"
+      "prov_wasGeneratedBy": {
+        "x_id": "niiri:6f6a8d7b-1697-418f-8286-b84f22325b5b"
       },
-      "rdfs:label": "Contrast Map: tone counting vs baseline"
+      "rdfs_label": "Contrast Map: tone counting vs baseline"
     },
     {
-      "@id": "niiri:a9cbc311-f192-44e1-b2ac-290dafdf4062",
-      "@type": [
+      "x_id": "niiri:a9cbc311-f192-44e1-b2ac-290dafdf4062",
+      "x_type": [
         "prov:Entity",
         "nidm_ExcursionSetMap"
       ],
-      "crypto:sha512": "d20b80683d8cf569e6c146bc4fe44506567004f1c0bf9b3571fa99e76e4f009146938a2b0b051198a46dbf7a6cd8d9701a20cf8eebe2e8f061216c1d772bb83e",
-      "dc:description": {
-        "@id": "niiri:d571cc3b-2ff7-47f6-93e5-79de687d806b"
+      "crypto_sha512": "d20b80683d8cf569e6c146bc4fe44506567004f1c0bf9b3571fa99e76e4f009146938a2b0b051198a46dbf7a6cd8d9701a20cf8eebe2e8f061216c1d772bb83e",
+      "dc_description": {
+        "x_id": "niiri:d571cc3b-2ff7-47f6-93e5-79de687d806b"
       },
-      "dct:format": "image/nifti",
-      "nfo:fileName": "ExcursionSet.nii.gz",
+      "dct_format": "image/nifti",
+      "nfo_fileName": "ExcursionSet.nii.gz",
       "nidm_inCoordinateSpace": {
-        "@id": "niiri:b8f4de59-d839-4f91-86a7-2697c1f65f44"
+        "x_id": "niiri:b8f4de59-d839-4f91-86a7-2697c1f65f44"
       },
-      "prov:atLocation": {
-        "@type": "xsd:anyURI",
-        "@value": "http://neurovault.org/collections/BNRPBNVV/fsl_default.nidm/ExcursionSet.nii.gz"
+      "prov_atLocation": {
+        "x_type": "xsd:anyURI",
+        "x_value": "http://neurovault.org/collections/BNRPBNVV/fsl_default.nidm/ExcursionSet.nii.gz"
       },
-      "prov:wasGeneratedBy": {
-        "@id": "niiri:9833881f-a909-48dd-8930-a832048c9cfe"
+      "prov_wasGeneratedBy": {
+        "x_id": "niiri:9833881f-a909-48dd-8930-a832048c9cfe"
       },
-      "rdfs:label": "Excursion Set Map"
+      "rdfs_label": "Excursion Set Map",
+      "nidm_hasMaximumIntensityProjection": {
```

Comment:
The json file for fsl_default should not have a MIP. It looks like the original file was overwritten.